#68 completed styling för about us page with responsive layout

ppaull123 opened the pull request:
Completes the “Om oss” page (#68) with HTML, styling and basic responsive layout.
Includes opening hours, FAQ section and placeholder content.
<img width="960" height="5978" alt="kino - about us - 480px" src="https://github.com/user-attachments/assets/d23eb4c1-ae72-40e9-aa1f-662bc5fa7957" />
<img width="2400" height="5442" alt="kino - about us - 1200px" src="https://github.com/user-attachments/assets/e5c0ac4a-326e-4a0a-bff5-50b2ad30a96c" />

ImanBatyah reviewed (commented):
Really nice work on the “About Us” page! 🙌                                                               You’ve clearly put effort into both structure and styling, and the layout feels well thought-out and responsive. Big plus for presenting the opening hours in a clean grid and using <details>/<summary> for the FAQ, that’s a solid, semantic, and accessible approach.

What’s especially good:
- Clear hero/title section,  the page feels visually “complete”.
- Opening hours for both the Cinema and the Bistro are nicely separated and easy to scan.
- The FAQ solution with details/summary is a great semantic choice.
- Good job adding aria-labels to the social links.

Small improvements (quick wins) before merge:
1) Alt text: I noticed some images have empty alt (alt=""). If the images are informative, consider adding more descriptive alt text (e.g., “Cinema auditorium”, “Bistro interior”) to improve accessibility.
2) Script tag: There’s a <script type="module" src=""></script> with an empty src. I’d either remove it or point it to the correct file (otherwise it may cause confusion later).
3) Placeholder content: Since the issue mentions body text is placeholder, you could make it clearer that it’s placeholder (or replace it with a short real text if you want to match the acceptance criteria more closely).
4) “Book a table” button: If it navigates to another page, it’s usually better to implement it as an <a> styled like a button for cleaner semantics and accessibility.

Overall: very strong PR 💪

VellaDona reviewed (approved):
In general the page looks very nice. Good job, Paul!
I approve the merge but before it is done I ask to make several changes:

- to style buttons same way as on the Start page to keep consistency;
-to  make the text about age limits more informative (you can take as a template the text in the Figma draft and alter it a bit; this is the real text that I copied from the competitor's website - Spegeln):

Åldersgränser:
7 år: Filmer med 7-årsgräns kan även ses av yngre barn i vuxens sällskap.  
11 år: Filmer med 11-årsgräns kan även ses av barn från 7 år i vuxens sällskap.  
15 år: Filmer med 15-årsgräns kan även ses av barn från 11 år i vuxens sällskap.  Med vuxen avses en person som har fyllt 18 år. Barn får se film på Biograf Spegeln i målsmans sällskap om filmens åldersgräns tillåter.  

I also have a question. How are the answers to FAQ shown without a JS file? Or you've just prepared them and a JS file will be the next step?

ppaull123 commented:
Update;
-styled the button to match the design used on start page.
-Updated the FAQ answer regarding age limits with more detailed and informative content.
-added ALT text to images.
-Clarified placeholders content for script tag and "Om oss" section.

@VellaDona for FAQ section there is no need for any JS, i used `<details>` and `<summary>` html elements that makes the job for openning and closing the questions/answers

nabeelnazeer13 commented on src/styles/_about-us.scss:
>    &__intro-title {
        font-size: 2rem;
        text-align: center;
        padding-top: 0.8rem;

        @media (min-width: $bp-medium) {
            font-size: 2rem;
            margin-bottom: 1rem;
            padding-top: 1rem;
        }
    }
    
    The font size is same for standard mobile layout and media query. if intended you can remove the second one

nabeelnazeer13 commented on src/styles/_about-us.scss:
```diff
@@ -0,0 +1,147 @@
+        height: auto;
+        display: block;
+        margin-bottom: 1.5rem;
+    }
```
You can use &__faq like you did above instead of vanilla CSS .about__faq

nabeelnazeer13 reviewed (approved):
I have now reviewed your code and I approve. I have a couple suggestions and point to raise but none of it is an error, bug or hindrance. Good job! It looks good and I approve so you can merge it
